app/overrides/spree/admin/users/edit/_add_reset_password_form.html.erb.deface

```diff
@@ -0,0 +1,20 @@
+<!--
+  insert_before "fieldset#admin_user_edit_api_key"
+  original "904c52ff702412d1dc8d55ff44d87d7f581f6675"
+-->
+
+<% if @user != try_spree_current_user %>
+  <fieldset class="no-border-bottom" data-hook="admin_user_reset_password">
+    <legend><%= I18n.t(:'spree.forgot_password') %></legend>
```

Comment:
@aitbw can you please remove `I18n`? It shadows `ActionView::Helpers::TranslationHelper#t`, see https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TranslationHelper.html